### PR TITLE
Update zh-CN.json

### PR DESCRIPTION
**Pull request by inblossoms**
Synchronize translation information to Chinese.

### PR Type

- [ ] Feature
- [ ] Bugfix
- [ ] Hotfix
- [x] Other (please describe):

### CODE REVIEW

**Comment by xuanxuan1231** [comment]
Repeated pull request #1296

File: locales/settings/zh-CN.json [hunk 1]
```diff
@@ -53,11 +53,16 @@
   "appearance.code_highlight_theme": "代码高亮主题",
   "appearance.content": "内容",
   "appearance.content_font": "正文字体",
+  "appearance.custom_css.button": "编辑",
+  "appearance.custom_css.description": "自定义上下文中的 css 样式",
```
Comment (Innei) [inline locales/settings/zh-CN.json:57]:
```suggestion
  "appearance.custom_css.description": "自定义条目渲染中的 CSS 样式",
```

File: locales/settings/zh-CN.json [hunk 2]
```diff
@@ -77,11 +82,17 @@
   "appearance.theme.label": "主题",
   "appearance.theme.light": "亮色",
   "appearance.theme.system": "跟随系统",
+  "appearance.thumbnail_ratio.description": "文章缩略图在列表中的比例",
```
Comment (Innei) [inline locales/settings/zh-CN.json:85]:
```suggestion
  "appearance.thumbnail_ratio.description": "列表中文章缩略图的比例",
```

**Review by Innei — CHANGES_REQUESTED** [review]

**Review by Innei — APPROVED** [review]
Thank you for your contribution.

**Pull request merged**